.github/workflows/deploy-book.yml

```diff
@@ -24,6 +24,12 @@ env:
   # By default, preprocess no branches ('').
   # Preprocessing no branches can also be set by ' '.
   BRANCHES_TO_PREPROCESS: ${{ vars.BRANCHES_TO_PREPROCESS }}
+  # If to copy the primary page to root
+  COPY: ${{ vars.COPY != '' && vars.COPY || 'false' }}
```

Comment:
Is copy the correct name? Would `REDIRECT` be better suited for the final behaviour?

Reply:
Yes, COPY is the intended name, though I maybe DO_COPY_PRIMARY would be better (?). Alternatively we could have a variable called e.g. BEHAVIOUR_PRIMARY with values either 'copy' or 'redirect'
```
    - name: Copy primary book to root, fail if names conflict
      timeout-minutes: 1
      if: ${{ env.COPY == 'true' }}
      run: |
        GLOBIGNORE=".:.."
        # -i leads to a prompt in case of conflict, => a timeout
        if [ -d final/"$PRIMARY_BRANCH" ]; then
          cp -irv final/"$PRIMARY_BRANCH"/* final/
        fi
```

Reply:
I think that is better!
Copy would in that case refer to the symlinking which is actually copying right?

Reply:
With redirect as the standard behaviour

Reply:
> I think that is better! Copy would in that case refer to the symlinking which is actually copying right?

Not quite -- currently symlinks are only used for aliases, and the primary branch is just copied over

Reply:
I see, let's do it as you suggested,  BEHAVIOUR_PRIMARY with values either 'copy' or 'redirect'